### DISCUSSION TITLE
Official version code name

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#457] [Ideas]
The official version has a code name, but not in the beta version. such as Mac, Linux Mite.
E.g 
Insect name
1.x  Antlion
2.x Bean beetle
3.x Comma butterfly

**Comment by probonopd** [comment]
[Codenames](https://github.com/helloSystem/hello/wiki/Codenames) had been defined but not widely used. If there is interest in this, we could use them more...

**Reply by louies0623** [reply]
I would prefer to name it after an insect because it makes one wonder how many species of insects there are on this planet that we don't know about.
And, I think it is better to correspond to the code from A-Z.
![image](https://user-images.githubusercontent.com/44593430/210285429-bfb75d15-5218-43f9-9486-ab684a91a2d3.png)
![image](https://user-images.githubusercontent.com/44593430/210285546-7d1f854a-30bd-4bde-88c8-4d2843f50902.png)

**Comment by louies0623** [comment]
I accidentally heard a podcast recently. The content talks about every scientist, physicist, mathematician and entrepreneur who has made positive contributions to mankind. Maybe it can be used as a code name, such as
system 1: Newton
2 : Boyle
3 : Galilei
4 : Pasteur 
5 : Planck
Lister
Maxwell
Crowell
HJ Heinz
Kopernik
Braun
Brewster
Morse
Kepler
Nightingale
Gutenberg
Descartes
Schickard
Da Vinci
Sklodowska-Curie
Euler
Einstein
Rosalind Elsie Franklin
Gaudi
Hertz
Michelangelo
Ohm
Ampere
Mendel
Joule
Thomson
Maury
Raffaello
Ronald Ross
Pascal
Cauchy
Helmholtz
Ieoh Ming Pei
George Washington Carver
George Gabriel Stokes
Booker T. Washington

**Comment by probonopd** [comment]
https://github.com/helloSystem/hello/wiki/Codenames :)
